{"name": "test_user_updates_task", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=134.0.6998.88)\nStacktrace:\n#0 0x55d188e5aa1a <unknown>\n#1 0x55d188912390 <unknown>\n#2 0x55d188909518 <unknown>\n#3 0x55d1888f9a39 <unknown>\n#4 0x55d1888fb73d <unknown>\n#5 0x55d1888f9dce <unknown>\n#6 0x55d1888f9775 <unknown>\n#7 0x55d1888f9449 <unknown>\n#8 0x55d1888f7159 <unknown>\n#9 0x55d1888f7a2a <unknown>\n#10 0x55d188915829 <unknown>\n#11 0x55d1889b0d85 <unknown>\n#12 0x55d188989bd2 <unknown>\n#13 0x55d1889b007b <unknown>\n#14 0x55d1889899a3 <unknown>\n#15 0x55d18895560e <unknown>\n#16 0x55d188956dd1 <unknown>\n#17 0x55d188e20ddb <unknown>\n#18 0x55d188e24cbc <unknown>\n#19 0x55d188e08392 <unknown>\n#20 0x55d188e25834 <unknown>\n#21 0x55d188dec1ef <unknown>\n#22 0x55d188e49038 <unknown>\n#23 0x55d188e49216 <unknown>\n#24 0x55d188e59896 <unknown>\n#25 0x7f1cbb2771f5 <unknown>", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"1247d9b66ecb93a968b3f3cdd858abcb\")>\n\n    def test_user_updates_task(driver):\n        username = \"kate11\"\n        password = \"kate11\"\n    \n        with allure.step(\"Navigate to the login page and authenticate\"):\n            user_on_tasks = LoginPage(driver)\n            user_on_tasks.get_login_page()\n>           user_on_tasks.authenticated_user(username, password)\n\ntests/ui/ui_tests/test_manage_tasks.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/ui/pages/login_page.py:32: in authenticated_user\n    self.open_url(\"http://localhost:5000/login\")\ntests/ui/pages/base_page.py:12: in open_url\n    self.driver.get(url)\n/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:355: in get\n    self.execute(Command.GET, {\"url\": url})\n/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:346: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f58e27f9460>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55d188e49216 \\\\u003Cunknown>\\\\n#24 0x55d188e59896 \\\\u003Cunknown>\\\\n#25 0x7f1cbb2771f5 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_SHADOW_ROOT:\n            exception_class = NoSuchShadowRootException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif (\n            status in ErrorCode.INVALID_SELECTOR\n            or status in ErrorCode.INVALID_XPATH_SELECTOR\n            or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER\n        ):\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=134.0.6998.88)\nE       Stacktrace:\nE       #0 0x55d188e5aa1a <unknown>\nE       #1 0x55d188912390 <unknown>\nE       #2 0x55d188909518 <unknown>\nE       #3 0x55d1888f9a39 <unknown>\nE       #4 0x55d1888fb73d <unknown>\nE       #5 0x55d1888f9dce <unknown>\nE       #6 0x55d1888f9775 <unknown>\nE       #7 0x55d1888f9449 <unknown>\nE       #8 0x55d1888f7159 <unknown>\nE       #9 0x55d1888f7a2a <unknown>\nE       #10 0x55d188915829 <unknown>\nE       #11 0x55d1889b0d85 <unknown>\nE       #12 0x55d188989bd2 <unknown>\nE       #13 0x55d1889b007b <unknown>\nE       #14 0x55d1889899a3 <unknown>\nE       #15 0x55d18895560e <unknown>\nE       #16 0x55d188956dd1 <unknown>\nE       #17 0x55d188e20ddb <unknown>\nE       #18 0x55d188e24cbc <unknown>\nE       #19 0x55d188e08392 <unknown>\nE       #20 0x55d188e25834 <unknown>\nE       #21 0x55d188dec1ef <unknown>\nE       #22 0x55d188e49038 <unknown>\nE       #23 0x55d188e49216 <unknown>\nE       #24 0x55d188e59896 <unknown>\nE       #25 0x7f1cbb2771f5 <unknown>\n\n/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:245: WebDriverException"}, "steps": [{"name": "Navigate to the login page and authenticate", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=134.0.6998.88)\nStacktrace:\n#0 0x55d188e5aa1a <unknown>\n#1 0x55d188912390 <unknown>\n#2 0x55d188909518 <unknown>\n#3 0x55d1888f9a39 <unknown>\n#4 0x55d1888fb73d <unknown>\n#5 0x55d1888f9dce <unknown>\n#6 0x55d1888f9775 <unknown>\n#7 0x55d1888f9449 <unknown>\n#8 0x55d1888f7159 <unknown>\n#9 0x55d1888f7a2a <unknown>\n#10 0x55d188915829 <unknown>\n#11 0x55d1889b0d85 <unknown>\n#12 0x55d188989bd2 <unknown>\n#13 0x55d1889b007b <unknown>\n#14 0x55d1889899a3 <unknown>\n#15 0x55d18895560e <unknown>\n#16 0x55d188956dd1 <unknown>\n#17 0x55d188e20ddb <unknown>\n#18 0x55d188e24cbc <unknown>\n#19 0x55d188e08392 <unknown>\n#20 0x55d188e25834 <unknown>\n#21 0x55d188dec1ef <unknown>\n#22 0x55d188e49038 <unknown>\n#23 0x55d188e49216 <unknown>\n#24 0x55d188e59896 <unknown>\n#25 0x7f1cbb2771f5 <unknown>\n\n", "trace": "  File \"/app/tests/ui/ui_tests/test_manage_tasks.py\", line 81, in test_user_updates_task\n    user_on_tasks.authenticated_user(username, password)\n  File \"/app/tests/ui/pages/login_page.py\", line 32, in authenticated_user\n    self.open_url(\"http://localhost:5000/login\")\n  File \"/app/tests/ui/pages/base_page.py\", line 12, in open_url\n    self.driver.get(url)\n  File \"/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 355, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1742462384781, "stop": 1742462385094}], "start": 1742462384781, "stop": 1742462385115, "uuid": "03effd93-a22e-4e59-a8a0-f40ea3c54b01", "historyId": "70a0db2353557475bd6623fe565ebf4c", "testCaseId": "70a0db2353557475bd6623fe565ebf4c", "fullName": "tests.ui.ui_tests.test_manage_tasks#test_user_updates_task", "labels": [{"name": "parentSuite", "value": "tests.ui.ui_tests"}, {"name": "suite", "value": "test_manage_tasks"}, {"name": "host", "value": "b248a0ba0ead"}, {"name": "thread", "value": "19-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.ui_tests.test_manage_tasks"}]}